tests/cf-local.spec.ts

```diff
@@ -460,15 +500,15 @@ describe("cf-local unit tests", () => {
             expect(result).to.be.empty;
         });
 
-        it("there is '(current' in parentthesisPos", async () => {
+        it("ok:: there is '(current' in parentthesisPos", async () => {
             cliResult.stdout = "test modified (current  test";
             cliResult.error = "";
             cliMock.expects("execute").withExactArgs(["targets"], undefined, undefined).resolves(cliResult);
             const result = await cfLocal.cfGetTargets();
             expect(result).to.be.deep.equal([{ label: "test modified", isCurrent: true, isDirty: true }]);
         });
 
-        it("no '(current' in parentthesisPos", async () => {
+        it("ok:: no '(current' in parentthesisPos", async () => {
```

Comment:
what is this prefix "ok::" used for ?

Reply:
test description combined according to the next pattern : 'result':: 'description'-'optional params', where 
  'result' - 'ok' | 'fail' | 'exception'
  'description' - free text explains test purpose/case
  'optional params' - optional to different test cases 
In general, I intend to use this pattern in all unit test for giving more flexible options to filter/search different test units.